CHANGELOG.md

```diff
@@ -1,5 +1,26 @@
 # CHANGELOG
 
+## v0.15.0 - 2021-06-29
+
+Fixes:
```

Comment:
```suggestion
### Fixes
```

Reply:
I'm following the previous markup in the changelog.

We can change it, but I'd make that a separate PR and update the entire file.

Reply:
Alright then